webcheck.go

```diff
@@ -204,15 +218,18 @@ type httpClientAndError struct {
 	Err error
 }
 
-func (fm *Frontman) newHTTPClientWithCustomMaxRedirectsLimit(maxRedirects int) *httpClientAndError {
-	if fm.httpTransport == nil {
-		fm.initHttpTransport()
+func (fm *Frontman) newClientWithOptions(
```

Comment:
Is there a need why this function is part of `Frontman `?

Reply:
Why is there a line-break after `(`?

Reply:
> Is there a need why this function is part of `Frontman `?

It was there already, but I guess this is just a http.Client config modifier, it's too small to be somewhere in shared package.

Reply:
> Why is there a line-break after `(`?

It's just me, I break very long lines. I can avoid doing that. Do you have a style guide to share with me? :)

Reply:
It's okay to break lines. https://github.com/golang/go/wiki/CodeReviewComments#line-length

But I think here is not the case because the line actually is not long.
Also, if you break line in the function declaration it's better to break line after each parameter, else it will look awkwardly.

Reply:
yeah, it was no argument against line breaking in general, just didn't seem necessary 👍